internal/app/mapper/mapper_mock.go

```diff
@@ -0,0 +1,21 @@
+package mapper
```

Comment:
can this live in the mock package too?

Reply:
possibly but this would be a deviation from how mocks in `upstream` are structured

Reply:
What is the idea behind the `mock` package?

Reply:
https://github.com/envoyproxy/xds-relay/pull/46#discussion_r410441281

Reply:
That said I'm not sure I understand the `*_mock.go` files being separate. Maybe @jyotimahapatra can elaborate?

Reply:
Isn't the conclusion of that thread that mocks live in the same package as the structures that they are mocking only in a different (called `mock`)?

Reply:
`client_mock` doesn't live under `mock` though

Reply:
I guess what I'm trying to say is that we don't need a package called `mock`. We've been putting mocks under the `mock` as a subdirectory of the structure they are mocking, but under the same package.